<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageForm</class>
 <widget class="QWidget" name="MessageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Отправка сообщения</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>отмена</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Адрес получателя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recipientLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Тема письма</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="subjectLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>отправить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
